tracer/dlist.mli

```diff
@@ -0,0 +1,21 @@
+(** Doubly linked list with all operations constant time *)
```

Comment:
I would have just created a ring of fixed size backed by an array. However if this already works, there's no need to change it.